src/ZoneServer/Scripting/AI/AiScript.Routines.cs

```diff
@@ -291,7 +291,7 @@ protected IEnumerable Follow(ICombatEntity followTarget, float minDistance = 50,
 
 				if (catchUp)
 				{
-					var closePos = this.Entity.Position.GetRelative(followTarget.Position, 50);
+					var closePos = followTarget.Position;
```

Comment:
GetRelative() is not correct when we use pathfinding now, because the closest position may no longer be within a straight line to the master/target.

Furthermore, isTargetInRange is already finding the distance to target/master so the follower stops moving at 50 distance either way, so we don't need any additional code change.